### PR TITLE
Texts for UM-039 (Aksumite collection)

### PR DESCRIPTION
**Pull request by eu-genia**
https://github.com/BetaMasaheft/Documentation/issues/1568
(more had to be created as they are not the same as the ones in Senodos, even when have similar labels)

### CODE REVIEW

**Comment by eu-genia** [comment]
@PietroLiuzzo out of curiousity - I have now created the pull request and see on top that I "added 28 commits **11 days ago**" - how can it be if most were added between yesterday and today? not very important but wanted to understand how this works.

**Comment by PietroLiuzzo** [comment]
you probably have created the branch before updating the master? to understand how this work I suggest one of the many tutorials available online, they know certainly better than me!

File: new/LIT6239EcclCanQ.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6239EcclCanQ" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Ecclesiastical canons</title>
```
Comment (thea-m) [inline new/LIT6239EcclCanQ.xml:8]: en

File: new/LIT6244Gangra.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6244Gangra" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Council of Gangra</title>
```
Comment (thea-m) [inline new/LIT6244Gangra.xml:8]: en, also in later records

**Review by thea-m — APPROVED** [review]
just some missed languge updates in the titles

**Pull request merged**